gpu_bdb/bdb_tools/utils.py

```diff
@@ -602,9 +613,16 @@ def verify_results(verify_dir):
     """
     verify_dir: Directory which contains verification results
     """
-    import cudf
-    import dask_cudf
-    import cupy as cp
+    if os.getenv("CPU_ONLY") == 'True':
+        import pandas as cudf
+        import dask.dataframe as dask_cudf
+        import numpy as cp
+    else:
+        import cudf
+        import dask_cudf
+        import cupy as cp
+    
+    
```

Comment:
Lets tackle this in a follow up PR as this is not query related .  As we are currently running on GPU machines we should be able to verify them.  
```suggestion
   
```